packages/gatsby-theme-store/src/components/SearchSuggestions/index.tsx

```diff
@@ -3,19 +3,24 @@ import type { FC } from 'react'
 
 import SuspenseDevice from '../Suspense/Device'
 import SearchSuggestionsProvider from './base/Provider'
+import { useRegion } from '../useRegion'
 
 const SearchSuggestionMobile = lazy(() => import('./Mobile'))
 const SearchSuggestionDesktop = lazy(() => import('./Desktop'))
 
-const SearchSuggestions: FC = () => (
-  <SearchSuggestionsProvider>
-    <SuspenseDevice device="mobile" fallback={null}>
-      <SearchSuggestionMobile />
-    </SuspenseDevice>
-    <SuspenseDevice device="desktop" fallback={null}>
-      <SearchSuggestionDesktop />
-    </SuspenseDevice>
-  </SearchSuggestionsProvider>
-)
+const SearchSuggestions: FC = () => {
+  const regionContext = useRegion()
```

Comment:
I think this hook should be closer to the place where is being used which is `useProductsSuggestions`.

Why is it not there?

Reply:
Not really, I thought of having it here in case of needing to use it somewhere besides the one that I'm using in a near future (the need of having the region id in other queries from the suggestions, for instance.)